docs/references/configuration.rst

```diff
@@ -158,6 +158,7 @@ db-extra-search-path        String  public            Y          PGRST_DB_EXTRA_
 db-max-rows                 Int     ∞                 Y          PGRST_DB_MAX_ROWS                 pgrst.db_max_rows
 db-plan-enabled             Boolean False             Y          PGRST_DB_PLAN_ENABLED             pgrst.db_plan_enabled
 db-pool                     Int     10                           PGRST_DB_POOL
+db-pool-max-idletime        Int     30                           PGRST_DB_POOL_MAX_IDLETIME
 db-pool-acquisition-timeout Int     10                           PGRST_DB_POOL_ACQUISITION_TIMEOUT
 db-pool-max-lifetime        Int     1800                         PGRST_DB_POOL_MAX_LIFETIME
```

Comment:
```suggestion
db-pool-acquisition-timeout Int     10                           PGRST_DB_POOL_ACQUISITION_TIMEOUT
db-pool-max-idletime        Int     30                           PGRST_DB_POOL_MAX_IDLETIME
db-pool-max-lifetime        Int     1800                         PGRST_DB_POOL_MAX_LIFETIME
```

Now that I notice, some of them aren't in alphabetical order anymore. May need a separate fix later.